docs/src/guides/spring/deploy/configure.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Configure Spring for Platform.sh"
+sidebarTitle: "Configure"
+weight: -100
+toc: false
+description: |
+    Review the basics of what makes up a Platform.sh project, including its three principle configuration files and how to define them for Spring.
+---
+
+{{< guides/config-desc name="Spring" >}}
+
+## Requests configuration: `routes.yaml`
+
+{{< guides/config-routes template="spring" name="Spring" >}}
```

Comment:
This is failing since we don't have a plain `spring` template: https://github.com/platformsh-templates?q=spring&type=&language=